cpp/CMakeLists.txt

```diff
@@ -88,10 +88,18 @@ else()
   set(ARROW_FULL_SO_VERSION "${ARROW_SO_VERSION}.${ARROW_VERSION_PATCH}.0")
 endif()
 
+set(ARROW_FORMAT_VERSION "1.0.0")
+set(ARROW_FORMAT_VERSION_MAJOR "1")
+set(ARROW_FORMAT_VERSION_MINOR "0")
+set(ARROW_FORMAT_VERSION_PATCH "0")
```

Comment:
What about extracting major/minor/patch from version string?
https://github.com/apache/arrow/blob/master/cpp/cmake_modules/FindNumPy.cmake#L83-L87
https://stackoverflow.com/a/18658684/6837980

Reply:
That works, thanks!